packages/api/src/util.ts

```diff
@@ -233,6 +235,8 @@ export function getEnv(): BackendEnv {
     reminderTaskHanderUrl: parse('REMINDER_TASK_HANDLER_URL'),
     integrationTaskHandlerUrl: parse('INTEGRATION_TASK_HANDLER_URL'),
     textToSpeechTaskHandlerUrl: parse('TEXT_TO_SPEECH_TASK_HANDLER_URL'),
+    textToSpeechName: parse('TEXT_TO_SPEECH_QUEUE_NAME'),
+    textToSpeechLocation: parse('TEXT_TO_SPEECH_QUEUE_LOCATION'),
```

Comment:
I also wonder if we can get this from our current location, or if the API defaults to that. We always want all infra in the same location, so maybe we don't need to specify.

Reply:
I looked at what google provides and I guess not. But maybe we could have a single `GCP_LOCATION`

Reply:
Haha. Yes, I also just looked at `cloud_tasks_client.ts`...
Yeah, I agree to hardcode these configs. I was thinking about having all these configs as env var is for self-hosting.

Reply:
yeah no real need to have lots of secrets.

Reply:
Thanks!